docs/download.html

```diff
@@ -0,0 +1,99 @@
+      <a href="index.html"><img src="images/test.png" style="width:100px;height:75px;"></a>
+        <a class="navbar-brand" href="index.html">Raider Planner</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="contribute.html">Contribute</a>
+            </li>
+            <li class="nav-item active">
+              <a class="nav-link" href="download.html">Download
+              <span class="sr-only">(current)</span>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    
+    <!-- This will explain how to download and start the program, however, we do not have a server nor do we have a file ready to automatically start the program-->
+    
+    <section>
+      <div class="containerVid">    
+          <div class="embededVideo">
+            <p> Watch this short video on the RaiderPlanner's current features:</p>
+            <iframe width="800" height="450" src="https://www.youtube.com/embed/gR8NEa77doI" frameborder="0" allowfullscreen></iframe>
+          </div>
+      </div>
+    </section>
+    
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6 order-2">
+            <div class="p-5">
+            <a href="images/download.png" download="RaiderPlanner-master.zip">
+              <img src="images/download.png" alt="" style="width:80%; height:10%; ">
+            </a>
+            </div>
+          </div>
+          <div class="col-md-6 order-1">
+            <div class="p-5">
+              <h2 class="display-3">Getting started:</h2>
+              <p>1. <a href="download.html" download="RaiderPlanner-master.zip">Download the file.</a> This link is not a working link right now.</p>
+              <p>2. Extract the file into your desired location.</p>
+              <p>3. Start up Eclipse and import your file.</p>
+              <p>4. Run RaiderPlanner.git to start up the program.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="py-3 bg-dark">
+      <div class="footer">
+        <p> Would you like to contribute to this project? Click <a href="contribute.html">HERE</a> to find out more information </p>
+        <p class="m-0 text-center text-white">Copyright &copy; Raider Planner 2017</p>

```

Comment:
RaiderPlanner is not an organization, so this should rather say "RaiderPlanner Contributors" and then link to the AUTHORS file in the project repository. I have just now added the file.